modules/json/format-json.c

```diff
@@ -83,6 +83,23 @@ tf_json_prepare(LogTemplateFunction *self, gpointer s, LogTemplate *parent,
       value_pairs_add_transforms(state->vp, vpts);
     }
 
+  if (cfg_is_config_version_older(parent->cfg, VERSION_VALUE_4_0) &&
```

Comment:
Shouldn't we use `FEATURE_TYPING_MIN_VERSION` here?

Reply:
the same applies as with the value-pairs case and I guess the afmongodb case is again the same.